tests/WordTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+require('../Word.php');
+use PHPUnit\Framework\TestCase;
+
+class WordTest extends TestCase {
+
+    public function testWordCardCreator() 
```

Comment:
this function should be called test_success_createWordCard

weird casing but it's because you can then use this cool command:

```
phpunit WordTest.php --testdox
```